### DISCUSSION TITLE
Where's the POD template?

### DISCUSSION THREAD

**Discussion by cybercoder** [cloudnative-pg/cloudnative-pg#4087] [General]
I need to set ```tolerations``` and ```affinity```. But I can't find it in the full example.

**Comment by gbartolini** [comment]
Hi @cybercoder,

Did you try searching in the documentation like in the image below?

![image](https://github.com/cloudnative-pg/cloudnative-pg/assets/182151/f28d5b60-01a5-44d1-b8b6-b9dab3bd2751)

Thanks,
Gabriele

P.S.: We don't provide yet the pod templates interface
